boskos/janitor/janitor.go

```diff
@@ -30,16 +30,18 @@ import (
 )
 
 var (
-	bufferSize  = 1 // Maximum holding resources
-	rTypes      common.CommaSeparatedStrings
-	poolSize    int
-	janitorPath = flag.String("janitor-path", "/bin/gcp_janitor.py", "Path to janitor binary path")
-	boskosURL   = flag.String("boskos-url", "http://boskos", "Boskos URL")
+	bufferSize      = 1 // Maximum holding resources
+	rTypes          common.CommaSeparatedStrings
+	poolSize        int
+	updateFrequency time.Duration
+	janitorPath     = flag.String("janitor-path", "/bin/gcp_janitor.py", "Path to janitor binary path")
+	boskosURL       = flag.String("boskos-url", "http://boskos", "Boskos URL")
 )
 
 func init() {
 	flag.Var(&rTypes, "resource-type", "comma-separated list of resources need to be cleaned up")
 	flag.IntVar(&poolSize, "pool-size", 20, "number of concurrent janitor goroutine")
+	flag.DurationVar(&updateFrequency, "update-freqency", 5*time.Minute, "How often to heartbeat owning resources.")
```

Comment:
should this be one minute per @sebastienvas ?

Reply:
I'd rather not throttle the server :-)

Reply:
reaper ttl (https://github.com/kubernetes/test-infra/blob/master/boskos/reaper/reaper.go#L31) is at 30min by default, so 5min update interval is enough.

Reply:
SGTM